checkov/main.py

```diff
@@ -415,6 +425,40 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
     return None
 
 
+def report_contributor_metrics(repository: str) -> None:  # ignore: type
```

Comment:
I think it might be worth extracting it to a different module, IMO it's not too related to the main script